api/authorizers/site.js

```diff
@@ -56,7 +56,26 @@ function authorizeFederalistUsersAdmin(user) {
 }
 
 // create is allowed for all
-const create = () => Promise.resolve();
+const create = async (user, siteParams) => {
+  const { organizationId } = siteParams;
+  const organizations = await Organization.forUser(user).findAll();
+
```

Comment:
definitely some personal preference but might be simpler to have
```
if (organizations.length === 0) return undefined;
```
here, then you don't have to keep checking it below.

Reply:
If they are not apart of any organizations, should we also make sure they do not specify an `organizationId`? 

```
if (organization.length === 0 && !organizationId) return undefined;
```